zulip/integrations/matrix/matrix_bridge.py

```diff
@@ -82,10 +82,15 @@ def _matrix_to_zulip(room: Any, event: Dict[str, Any]) -> None:
 def get_message_content_from_event(event: Dict[str, Any]) -> str:
     irc_nick = shorten_irc_nick(event['sender'])
     if event['type'] == "m.room.member":
+        return
+        # uncomment the lines below and delete the above `return`
+        # to display joining and leaving messages
+        """
```

Comment:
I would use a Python code comment, not a block string, for this.

Reply:
(Also, might be better to just make it a command-line flag in the tool)

Reply:
Added a command-line flag! re: https://github.com/zulip/python-zulip-api/pull/396/commits/1556cb07e2d133f020cc56ce934423f0864c1c7e